### ISSUE TITLE
The meaning of some parameters

### ISSUE THREAD

**Issue by NanjiaHan** [salesforce/densecap#2]
Hi @LuoweiZhou ，when I read the code, I find these parameters in the "action_prop_dense_cap.py": scst_loss, gated_mask, and learn_mask.
But I don't know what they mean. Can you explain in more detail?  Thank you.

**Comment by LuoweiZhou** [comment]
Hi Nanjia, please see the followings.
i) `scst_loss` indicates self-critical sequence training (as in https://arxiv.org/abs/1612.00563). We didn't report results w/ this training loss and hence it's deprecated. Still, we keep this option out there in case people need (might need to upgrade some of the code to pytorch 0.4)
ii) `learn_mask` in the `test.py` indicates the mask is learned from the data, in opposite to the non-differentiable mask as mentioned at the end of Sec. 4.4
iii) further, `gate_mask`indicates the learned mask is gated by the proposal confidence score, as in Eq. 17

**Comment by NanjiaHan** [comment]
Oh! Now I understand.Thanks for your reply.

**Comment by hollowgalaxy** [comment]
Hello, what is the `pos_thresh` argument? Is the threshold for when the start time happens? (based on [this](https://github.com/salesforce/densecap/blob/9eae43ea4819a169c719208c5262a834e20e6b34/model/action_prop_dense_cap.py#L391))

**Comment by LuoweiZhou** [comment]
@hollowgalaxy In the event proposal module, a proposal is treated as a positive sample during training if it has over `pos_thresh` with an arbitrary GT segment (hence classified as 1 vs. 0 for negative samples).
